core/conversion/conversion.cpp

```diff
@@ -265,6 +269,7 @@ void MarkOutputs(ConversionCtx* ctx, at::ArrayRef<const torch::jit::Value*> outp
       }
 
       if (!setOutput) {
+        out_tensor->setType(util::ScalarTypeToTRTDataType(out_types[out_idx++]));
```

Comment:
The out_idx needs to be incremented even if one of the outputs is an input (see where setOutput may be set above)